includes/css/scss/_module-header-brand-dealership.scss

```
@@ -7,24 +7,13 @@
 		content: "";
 		height: 2px;
 		width: calc((100% - #{$container-width}) / 2);
```

No need for `calc` here
```suggestion
		flex: 1;
```